.github/workflows/build.yml

```diff
@@ -29,7 +29,9 @@ jobs:
           pip install -r requirements.txt
       - name: Check style against standards using prospector
         shell: bash -l {0}
-        run: prospector -o grouped -o pylint:pylint-report.txt
+        run: |
+            pip install prospector[with_pyroma]
```

Comment:
Perhaps you could add a `requirements-test.txt` where you specify you dependencies for testing. Here you can then also fix versions.